lib/internal/streams/readable.js

```diff
@@ -196,7 +196,8 @@ function Readable(options) {
   Stream.call(this, options);
 
   destroyImpl.construct(this, () => {
-    maybeReadMore(this, this._readableState);
+    if (!this.isPaused())
```

Comment:
try `this[kPaused] !== false`

Reply:
I wonder if I shouldn't simply change `isPaused()`, no one is using, there is one occurrence in `net.afterConnect` but this won't affect it

Reply:
Maybe `isPaused` should be deprecated

Reply:
isPaused is part of public API and can't be easily changed or removed

Reply:
What happens if we just simply remove maybeReadMore from here?